examples/seq2seq/finetune_trainer.py

```diff
@@ -86,28 +93,21 @@ class DataTrainingArguments:
             "than this will be truncated, sequences shorter will be padded."
         },
     )
-    max_target_length: Optional[int] = field(
+    max_length: Optional[int] = field(
```

Comment:
Why was this done in the last second? This and all the renames below just broke all the scripts that were relying on --max_target_length --val_max_target_length args.

Reply:
The discussion and the agreement as how I understood it was to rename only the new arguments you added and leaving everything else as before. We need source and target for datasets.

Reply:
Sorry, I should have been a bit more careful at my second review of the PR...I'd probably prefer replacing all `--val_max_target_length` and `test_max_target_length` with `--max_length`, but I'm also fine with reverting the change on `DataTrainingArguments`

Reply:
Thank you for clarifying, so indeed you didn't not suggest to change the cl args as I read it.

I guess @sgugger read your mind then - I have no problem with that other than needed a deprecation cycle for such changes because it really interferes with my work at integrating / benchmarking / testing deepspeed and fairscale.

Then if we are now switching to refactoring the cl args - there should be no `eval_max_length` either, no?

Reply:
IMO, one `max_length` should be enough. It does not make much sense to tokenizer longer than the model is able to generate and the other way around would result in errors. So IMO the best approach is to have one `max_length` referring to the maximum length that the model should be able to generate. 

The question then is whether we need a `max_source_length` or `max_seq_length` as it's called in `run_mlm.py` or not. I think you guys have more experience here.